src/main/java/net/sf/jabref/logic/util/io/FileUtil.java

```diff
@@ -127,29 +126,20 @@
     /**
      * Copies a file.
      *
-     * @param source         File Source file
-     * @param dest           File Destination file
+     * @param pathToSourceFile         File Source file
+     * @param pathToDestinationFile           File Destination file
```

Comment:
Please remove unnecessary white spaces as they are ignored anyway when viewing the comment as javadoc.